annotated-pojos/src/test/java/annotated/pojos/PokemonQueryTest.java

```diff
@@ -57,7 +57,7 @@ public void testPokemonQuery() {
 
       QueryFactory queryFactory = Search.getQueryFactory(cache);
 
-      Query query = queryFactory.create("FROM annotated.pojos.Pokemon p where p.type1 = :type");
+      Query query = queryFactory.create("FROM pokemons.Pokemon p where p.type1 = :type");
```

Comment:
Why change the package name? To make a separation between java package names and proto package names?

Reply:
yes. also explained in the commit comment.